### PR TITLE
ticdc: add doc for bi-directional replication

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

as title.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/11712, https://github.com/pingcap/docs-cn/pull/11719
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -881,3 +884,44 @@ In this command:
...
+
+Starting from v6.4.0, TiCDC supports bi-directional replication among multiple TiDB clusters. Based on this feature, you can create a multi-master TiDB solution using TiCDC.
+
+### Limitations
+
+- Bi-directional replication clusters do not support replicating DDL. You need to manually execute all DDL statements in each TiDB cluster, and the table schemas before and after executing the DDL statement must be compatible.
+
+- When the bi-directional replication clusters are running, you cannot add or remove a cluster.
+
+- Bi-directional replication clusters cannot detect write conflict, which might cause undefined behavior. Therefore, you must ensure there is no write conflict from the application side.
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:898]:
```suggestion
- Bi-directional replication clusters cannot detect write conflicts, which might cause undefined behaviors. Therefore, you must ensure there are no write conflicts from the application side.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -595,6 +595,9 @@ ignore-txn-start-ts = [1, 2]
 # Filter syntax: https://docs.pingcap.com/tidb/stable/table-filter#syntax.
 rules = ['*.*', '!test.*']
 
+# Filter out data written by other changefeeds in the upstream. Set to `false` by default.
+ignore-rows-written-by-ticdc = false
+
```
Comment (xiongjiwei) [inline ticdc/manage-ticdc.md:600]: this change has been removed in cn

**Review by xiongjiwei — APPROVED** [review]

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (ran-huang) [inline ticdc/manage-ticdc.md:600]:
```suggestion
```

Comment (ran-huang) [reply ticdc/manage-ticdc.md:600]: done

File: system-variables.md [hunk 3]
```diff
@@ -835,6 +835,15 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - This variable is used to control whether to enable the [baseline capturing](/sql-plan-management.md#baseline-capturing) feature. This feature depends on the statement summary, so you need to enable the statement summary before you use baseline capturing.
 - After this feature is enabled, the historical SQL statements in the statement summary are traversed periodically, and bindings are automatically created for SQL statements that appear at least twice.
 
+### `tidb_cdc_write_source` <span class="version-mark">New in v6.5.0</span>
```
Comment (ran-huang) [inline system-variables.md:838]:
```suggestion
### tidb_cdc_write_source <span class="version-mark">New in v6.5.0</span>
```

**Review by xiongjiwei — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -881,3 +881,62 @@ In this command:
...
+Bi-directional replication does not support replicating DDL statements.
+
+If you need to execute DDL statements, take the following steps:
+
+1. Pause the write operations in the tables that need to execute DDL in all clusters. If the DDL statement is adding a non-unique index, skip this step.
+2. After the write operations of the correponding tables in all clusters have been replicated to other clusters, manually execute all DDL statements in each TiDB cluster.
+3. After the DDl statements are executed, resume the write operations.
+
+Note that a DDL statement that adds non-unique index does not break bi-direcional replication, so no need to pause the write operations in the corresponding table.
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:928]:
```suggestion
Note that a DDL statement that adds non-unique index does not break bi-direcional replication, so you do not need to pause the write operations in the corresponding table.
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -881,3 +881,62 @@ In this command:
...
+
+After the application has stopped writing data, you can insert a special record into each cluster. By checking the two special records, you can make sure that data in two clusters are consistent.
+
+After the check is completed, you can stop the changefeed to stop bi-direcional replication.
+
+### Limitations
+
+- For the limitations of DDL, see [Execute DDL](#execute-ddl).
+
+- Bi-directional replication clusters cannot detect write conflict, which might cause undefined behaviors. Therefore, you must ensure there are no write conflicts from the application side.
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:940]:
```suggestion
- Bi-directional replication clusters cannot detect write conflicts, which might cause undefined behaviors. Therefore, you must ensure there are no write conflicts from the application side.
```

**Pull request merged**